### PR TITLE
Access request page

### PR DESCRIPTION
**Pull request by larskendall**

### CODE REVIEW

File: src/components/pages/RequestAccessPage/RequestAccessPage.tsx [hunk 1]
```diff
@@ -1,9 +1,227 @@
...
+        {/* Requests */}
+        {accessRequests.length > 0 && (
+          <Paper className={classes.paper}>
+            <TableContainer>
+              <Table size="small" aria-label="access requests table">
+                <TableHead>
+                  <TableRow>
+                    <TableCell>Group Name</TableCell>
+                    <TableCell>Contact</TableCell>
+                    <TableCell style={{ paddingLeft: '28px' }}>Email</TableCell>
```
Comment (jdgarrett) [inline src/components/pages/RequestAccessPage/RequestAccessPage.tsx:100]: Hardcoded styling

File: src/components/pages/RequestAccessPage/RequestAccessPage.tsx [hunk 2]
```diff
@@ -1,9 +1,227 @@
...
+                <TableBody>
+                  {accessRequests.map(req => (
+                    <TableRow key={req.org.name}>
+                      <TableCell component="th" scope="row">
+                        {req.org.name}
+                      </TableCell>
+                      <TableCell>{`${req.org.contact.first_name} ${req.org.contact.last_name}`}</TableCell>
+                      <TableCell>
+                        <Tooltip title="Copy to clipboard" aria-label="copy email to clipboard">
+                          <IconButton onClick={() => handleEmailClick(req.org)} aria-label="email">
```
Comment (jdgarrett) [inline src/components/pages/RequestAccessPage/RequestAccessPage.tsx:114]: Isn't it normal practice to use a mailto: link for emails?

File: src/components/pages/RequestAccessPage/RequestAccessPage.tsx [hunk 3]
```diff
@@ -1,9 +1,227 @@
...
+              All Groups
+            </Typography>
+          </Toolbar>
+          <TableContainer>
+            <Table size="small" aria-label="all groups table">
+              <TableHead>
+                <TableRow>
+                  <TableCell>Group Name</TableCell>
+                  <TableCell>Contact</TableCell>
+                  <TableCell style={{ paddingLeft: '28px' }}>Email</TableCell>
```
Comment (jdgarrett) [inline src/components/pages/RequestAccessPage/RequestAccessPage.tsx:178]: Hardcoded styling

File: src/components/pages/RequestAccessPage/RequestAccessPage.tsx (hunk 2 above)
Comment (larskendall) [reply src/components/pages/RequestAccessPage/RequestAccessPage.tsx:114]: I was initially going to do that and then settled on copying the normal address to the clipboard for some reason. Maybe it's better to treat it like a regular email link though. I'll switch to that.

**Pull request merged**